docs/config-schema.md

```diff
@@ -139,6 +141,21 @@ The **nested** `collectd` config object has the following fields:
 
 
 
+## metricsToInclude
+The **nested** `metricsToInclude` config object has the following fields:
+
+
+
+| Config option | Required | Type | Description |
+| --- | --- | --- | --- |
+| `dimensions` | no | map of string | A map of dimension key/values to match against.  All key/values must match a datapoint for it to be matched. |
+| `metricNames` | no | list of string | A list of metric names to match against, OR'd together |
+| `metricName` | no | string | A single metric name to match against |
+| `monitorType` | no | string | (**Only applicable for the top level filters**) Limits this scope of the filter to datapoints from a specific monitor. If specified, any datapoints not from this monitor type will never match against this filter. |
+| `negated` | no | bool | Negates the result of the match so that it matches all datapoints that do NOT match the metric name and dimension values given. This does not negate monitorType, if given. (**default:** `false`) |
+
+
+
 ## metricsToExclude
```

Comment:
Yeah I see that...it is related to the fact that there is a `metricsToExclude` section in the main config as well as under each `monitor` list item.

Since they are exactly the same right now and the links from the config to the nested sections works fine I will not fix now.